lib/slack-invite.js

```diff
@@ -1,25 +1,35 @@
 
 import request from 'superagent';
+import util from 'util';
 
 export default function invite({ org, token, email, channel }, fn){
   let data = { email, token };
 
   if (channel) {
     data.channel = channel;
+    data.channels = channel;
     data.ultra_restricted = 1;
     data.set_active = true;
+    data._attempts = 1;
   }
 
+  console.log(`Requesting invite with data=${util.inspect(data)}`);
```

Comment:
Logging is done differently in the rest of the project. It's event-driven
https://github.com/rauchg/slackin/blob/master/lib/log.js